client/devdocs/async-load/placeholder.jsx

```diff
@@ -0,0 +1,37 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import PostItem from 'blocks/post-item';
+import SitePlaceholder from 'blocks/site/placeholder';
+
+export default class DevdocsAsyncLoadPlaceholder extends React.PureComponent {
+	static displayName = 'DevdocsAsyncLoadPlaceholder';
+	static propTypes = {
+		count: PropTypes.number.isRequired,
+	};
+
+	render() {
+		return (
+			<Main className="design async-load__devdocs-placeholder">
+				{ Array( this.props.count )
+					.fill( 1 )
```

Comment:
note that `Array.fill()` isn't supported in IE at all. I'm not sure if it's poly-filled but I somehow doubt it.

Reply:
Oh shoot, good call. I've been writing a lot of code in repos that polyfill stuff like that, I'll fix that up.